.storybook/guides/deprecation.mdx

```diff
@@ -0,0 +1,45 @@
+import { Meta } from '@storybook/blocks';
+
+<Meta title="Guides/Deprecation workflow"/>
+
+# Deprecating a component
+
+When a component is deprecated, it is no longer recommended for use. Though the component will be removed from the codebase and will no longer receive feature updates, we will apply bug fixes to the package as necessary. We will also provide a migration path to a recommended alternative.
+
+## Deprecation process
+
+**Deprecation of a component is typically initiated and communicated by the design team.** They will indicate that a component is going to be deprecated and identify reccomended alternatives and proposed language around migrating away from that component's use.
+
+### Developer responsibilities
+
+#### Due diligence
+
+Before starting a deprecation, connect with the Spectrum Web Components team to align on the timing of the deprecation. Make sure downstream users are aware that the component is being deprecated and have a migration path to the recommended alternative. Give downstream consumers time to express any concerns or blockers.
```

Comment:
This might not be a now thing, but I wonder if we should eventually flush out this process to make the notices more public since the repo is open source.

Reply:
Yeah for sure! I've not used discussions before but at a brief glance it looks kinda perfect for this.